UM/FileHandler/FileHandler.py

```diff
@@ -125,7 +125,7 @@ def addWriter(self, writer: "FileWriter") -> None:
     # \param file_name The name of the file to load.
     # \param kwargs Keyword arguments.
     # \returns None if nothing was found
-    def readerRead(self, reader: FileReader, file_name: str, **kwargs: Any):
+    def readerRead(self, reader, file_name: str, **kwargs: Any):
```

Comment:
Why no typing?

Reply:
The reason for this is that both the MeshFileHandlers and the WorkspaceFileHandlers inherit from this. But at that point, the WorkspaceFileHandler can't override the signature to accept a WorkspaceFileReader only, since it's not fully compatible.

This basicly left me with two options; Either remove this typing, or re-work the entire structure that it's in right now. After a lot of doubting, I decided to just remove the typing here. That being said, I'm open to suggestions as to how this can be resolved differently.